census/README.md

```diff
@@ -90,6 +90,12 @@ export OUTPUT_DIR=census_output
 rm -rf $OUTPUT_DIR
 ```
 
+### Model reuse
```

Comment:
If you do make the change to warning/error instead of deletion, it should also be reflected here.